tests/search/includes/classes/queue/test-class-cron.php

```diff
@@ -252,11 +252,11 @@ public function configure_concurrency_data() {
 			],
 			[
 				20,
-				[ 'vip_search_queue_processor' => 5 ],
+				[ 'vip_search_queue_processor' => 3 ],
 			],
 			[ // max of 5 takes over
```

Comment:
Minor nitpick, can we update the comment to reflect the change of 3?

Reply:
thanks for the catch, addressed!